README.md

```diff
@@ -1,55 +1,45 @@
-# Mission Control
+# **Mission Control**
```

Comment:
Don't bold headers; they already have a distinct style to them.

Reply:
This was a weird thing done on the other documentation for some reason, so I copied it. Personally yeah kind of redundant. Unbolded all headers